packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -42,14 +45,50 @@ describe("tooltip integration tests", () => {
         const anchor = anchorWrapper.getDOMNode();
         // start hovering
         anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseleave"));
         expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
 
+        const bubbleTimeoutCheck = (callback) =>
+            setTimeout(() => {
+                expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
+                callback && callback();
+            }, TooltipDisappearanceDelay / 2);
+
+        bubbleTimeoutCheck(callback);
```

Comment:
This timeout check seems completely separate from `Tooltip` and `TooltipBubble`.  Am I missing something?

Reply:
It looks like it waits a bit of time before doing the `expect`.  Since the code is based on timers, calling `jest.advanceTimersByTime(TooltipDisappearanceDelay / 2);` should be sufficient.